### ISSUE TITLE
No 'SYSTEM.CNF' file found on the cdrom0 device.

### ISSUE THREAD

**Issue by tokumeiwokiboushimasu** [jpd002/Play-#621]
Hi, in many games, this message appears and the games do not seem to boot.
Maybe regression of the below?
https://github.com/jpd002/Play-/issues/55

**Comment by Thunder07** [comment]
you'll need to provide more detail about the issue.
such as OS, Game, build version.

did you compile Play! yourself, if you're on a linux based system this might be a known issue, possibly duplicate of [this](https://github.com/jpd002/Play-/issues/609) issue

**Comment by tokumeiwokiboushimasu** [comment]
I compiled Play! by myself on Linux. So maybe the same issue, sorry.

**Comment by tokumeiwokiboushimasu** [comment]
The build version is 1b79f0d927c1c29794acc87720849db4e7d8f47d,
but this occur at least few days ago.
Games are for example 
 Curry House CoCo Ichibanya ntsc-j SLPS-20367
 Tetris - Kiwame Michi ntsc-j SLPM-62423
(Sorry, I only have minor games)

**Comment by tokumeiwokiboushimasu** [comment]
And these game images could boot on the past builds.

**Comment by jpd002** [comment]
Could this be related to #609 somehow?

**Comment by Thunder07** [comment]
@jpd002 it's the same issue, this is the error showed by the gui, while the other issue is the error that shows up in the console.

**Comment by jpd002** [comment]
Fix has been merged. Booting from ISO disc images on Linux should work fine now.

**Comment by happygreenfrog** [comment]
I don't know whether opening a new issue or commeanting on this one is the best way to deal with this, but... my attempts to use most 64-bit Windows builds, aside from, weirdly enough, "Play-20170411" (and possibly a few other builds from around that time, but definitely that one), gives me this same error when attempting to use my physical Blu-Ray drive to read the games. This issue happens in the current weekly build, if that helps.

Windows version is Windows 10 Home, Version 10.0.17134 Build 17134. The Blu-Ray drive is an external LG-CP40NG10 Slim Portable Blu-ray Multi Drive. The game I'm attempting to boot right now is Mega Man X Command Mission, but in the past when I attempted this, I got that error from pretty much every other game I tried too, so I'm not sure it is game-specific. Any other info that may be useful in figuring out what's going on?
